packages/cart/src/models/adjustment-line.ts

```diff
@@ -19,10 +19,10 @@ export default abstract class AdjustmentLine {
   @Property({ columnType: "text", nullable: true })
   promotion_id?: string | null
 
-  @Property({ columnType: "text" })
-  code: string
+  @Property({ columnType: "text", nullable: true })
+  code?: string | null
```

Comment:
```suggestion
  code: string | null = null
```

same everywhere for properties